client/src/components/HistoryViewer/HistoryViewer.scss

```diff
@@ -171,6 +171,28 @@
   }
 }
 
+.history-viewer__version-detail-diff {
+
+  del, ins {
+    border-radius: .192 rem;
+    padding: .192 rem;
+  }
+
+  del {
+    background-color: $red;
+    color: white;
+    border-radius: .192rem;
+    padding: .192rem;
+  }
+
+  ins {
+    background-color: $green;
+    color: white;
+    border-radius: .192 rem;
+    padding: .192 rem;
```

Comment:
not sure if the linter is picking this up, but you shouldn't have spaces between the number and the unit, also please remove the empty line on 175

Reply:
Neither `yarn lint` nor `yarn build` did...  But I have refactored that code anyway in the latest commit.